ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/avdtestbase.py

```diff
@@ -43,6 +46,35 @@ def render(self) -> dict:
         """
         return getattr(self, "test_definition", None) or {}
 
+    def get_host_struct_cfg(self, host: str) -> dict:
```

Comment:
can we rename the function name
```suggestion
    def get_host_structured_config(self, host: str) -> dict:
```

Reply:
Fixed in da322e1ab1e922b65e6353b1ab25e092f38d9bf3